pytest_timeout.py

```diff
@@ -433,6 +432,43 @@ def dump_stacks():
         write("".join(traceback.format_stack(frame)))
 
 
+def _getdimensions():
```

Comment:
I think this is more consistently named `get_terminal_dimensions()` given the other name (especially the consistency of when using underscores between words, the names are more bikesheddy and I don't mind as much).

Also no need to prefix either of these two functions with underscores here.  This module does not have a public API and we haven't been doing this for other functions (uh, lol, that's not true, but probably should have been true)